lib/internal/Magento/Framework/Amqp/Queue.php

```diff
@@ -73,11 +76,11 @@ public function dequeue()
         /** @var AMQPMessage $message */
         try {
             $message = $channel->basic_get($this->queueName);
-        } catch (AMQPProtocolConnectionException $e) {
+        } catch (AMQPTimeoutException $exception) {
```

Comment:
As I see `basic_get` could throw not only described in phpdoc `AMQPTimeoutException` exception. I believe that we should use `Exception` here